plotly/plotly/plotly.py

```diff
@@ -454,6 +458,29 @@ def __init__(self, stream_id):
         self.connected = False
         self._stream = None
 
+    def get_streaming_specs(self):
+        """
+        Returns the streaming server, port, ssl_enabled flag, and headers.
+
+        """
+        streaming_url = get_config()['plotly_streaming_domain']
+        ssl_enabled = 'https' in streaming_url
+        port = self.HTTPS_PORT if ssl_enabled else self.HTTP_PORT
```

Comment:
Nice. I think we should default to `443` instead of `80` from now on. so like `port = self.HTTP_PORT if 'http:' in streaming_url else self.HTTPS_PORT`